packages/react-instantsearch/src/connectors/connectMenu.js

```diff
@@ -28,7 +28,7 @@ function getValue(name, props, searchState) {
   return name === currentRefinement ? '' : name;
 }
 
-const sortBy = ['count:desc', 'name:asc'];
+const sortBy = ['isRefined', 'count:desc', 'name:asc'];
```

Comment:
What we could do is change this sort order only if SFFV is activated, then it's not a breaking change.